conf/app_version_v1_0.routes

```diff
@@ -1,5 +1,5 @@
-POST        /corporation-tax                uk.gov.hmrc.organisationsmatchingapi.controllers.MatchingController.matchOrganisationCt
-GET         /corporation-tax/:matchId       uk.gov.hmrc.organisationsmatchingapi.controllers.MatchingController.matchedOrganisationCt(matchId: java.util.UUID)
+POST        /matching/corporation-tax                uk.gov.hmrc.organisationsmatchingapi.controllers.MatchingController.matchOrganisationCt
```

Comment:
I don't think the routes should include the /matching/ part - I think it's added by the framework / environment. See individuals-matching.